webapi/SemanticKernelExtensions.cs

```diff
@@ -143,34 +141,19 @@ private static void AddSemanticTextMemory(this IServiceCollection services)
                     throw new InvalidOperationException("MemoriesStore type is AzureCognitiveSearch and AzureCognitiveSearch configuration is null.");
                 }
 
-                // ACS's vector search where users provide their embeddings
-                if (config.AzureCognitiveSearch.UseVectorSearch)
-                {
-                    services.AddSingleton<IMemoryStore>(sp =>
-                    {
-                        return new AzureSearchMemoryStore(config.AzureCognitiveSearch.Endpoint, config.AzureCognitiveSearch.Key);
-                    });
-                    services.AddScoped<ISemanticTextMemory>(sp => new SemanticTextMemory(
-                        sp.GetRequiredService<IMemoryStore>(),
-                        sp.GetRequiredService<IOptions<AIServiceOptions>>().Value
-                            .ToTextEmbeddingsService(logger: sp.GetRequiredService<ILogger<AIServiceOptions>>())));
-                }
-                // ACS's semantic search where ACS calculates the embeddings
-                else
+                services.AddSingleton<IMemoryStore>(sp =>
                 {
-                    services.AddSingleton<ISemanticTextMemory>(sp => new AzureCognitiveSearchMemory(config.AzureCognitiveSearch.Endpoint, config.AzureCognitiveSearch.Key));
-                }
+                    return new AzureCognitiveSearchMemoryStore(config.AzureCognitiveSearch.Endpoint, config.AzureCognitiveSearch.Key);
+                });
+                services.AddScoped<ISemanticTextMemory>(sp => new SemanticTextMemory(
```

Comment:
nit: move this out to reduce duplication.

Reply:
Done